lib/xsdParser.coffee

```diff
@@ -354,6 +369,14 @@ module.exports =
           memberType = @types[t]
           type.xsdChildren.push memberType.xsdChildren[0] if memberType
 
+      # If it's a list
+      else if type.xsdChildrenMode == 'list'
+        listType = @types[type.leftLabel]
+        if not listType
```

Comment:
If I remember correctly, the `types` dictionary doesn't contain the built-in types like `string`, so if it's a list of strings it will fail here, right?

Reply:
I will take a look at this. Good point.